detections/web/citrix_adc_and_gateway_citrixbleed_2_memory_disclosure.yml

```diff
@@ -0,0 +1,75 @@
+name: Citrix ADC and Gateway CitrixBleed 2 Memory Disclosure
+id: bef92f3f-7dc8-413a-8989-50581039e250
+version: 1
+date: '2025-01-07'
+author: Michael Haag, Splunk
+status: production
+type: TTP
```

Comment:
I'm in between putting this to TTP / Anomaly. Just because we dont have the post the data. Especially when this is fixed in envs and the endpoint is hit again.

What do you think @MHaggis ?